android/src/main/kotlin/br/com/oitiliveness3d/oiti_liveness3d/theme/Liveness3DTheme.kt

```diff
@@ -0,0 +1,175 @@
+package br.com.oitiliveness3d.oiti_liveness3d.theme
+
+import androidx.annotation.DrawableRes
+import br.com.oiti.liveness3d.R
+import br.com.oiti.liveness3d.theme.Liveness3DButtonLocation
+import br.com.oiti.liveness3d.theme.Liveness3DExitAnimationStyle
+import br.com.oiti.liveness3d.theme.Liveness3DTheme
+
+class Liveness3DTheme(
+    private val textsBuilder: Map<String, String?>?,
+    //Guidance customization
+) {
+    private val guidanceCustomizationBackgroundColors: String? = textsBuilder?.get("guidanceCustomizationBackgroundColors")
+    private val guidanceCustomizationForegroundColor: String? = textsBuilder?.get("guidanceCustomizationForegroundColor")
```

Comment:
Sugestão para descer o comentário

```suggestion
class Liveness3DTheme(
    private val textsBuilder: Map<String, String?>?,
) {
    //Guidance customization
    private val guidanceCustomizationBackgroundColors: String? = textsBuilder?.get("guidanceCustomizationBackgroundColors")
    private val guidanceCustomizationForegroundColor: String? = textsBuilder?.get("guidanceCustomizationForegroundColor")
```